allchblk.c

```diff
@@ -1005,6 +1005,12 @@ STATIC struct hblk *GC_allochblk_nth(size_t lb_adjusted, int k,
     return hbp;
 }
 
+# if defined(VALGRIND_TRACKING)
```

Comment:
`#ifdef VALGRIND_TRACKING` (fix this everywhere)

Reply:
Fixed